.circleci/config.yml

```diff
@@ -184,6 +185,14 @@ jobs:
       - image: circleci/mysql:5.7.26
         auth: &docker_auth
 
+  php74-tests:
+    <<: *php_job
+    docker:
+      - image: greenpeaceinternational/p4-unit-tests:php7.4
+        auth: &docker_auth
+      - image: circleci/mysql:5.7.26
+        auth: &docker_auth
```

Comment:
It seems that all these statements in this CI config are wrong

`Warning: No authentication provided, using CircleCI credentials for pulls from Docker Hub.`

The [correct](https://github.com/greenpeace/planet4-builder/blob/main/.circleci/config.yml#L11-L12) should be:

```
auth:
  <<: *docker_auth
```

Reply:
Nice catch :+1:  It was a mix of bad syntax and missing environment variables from org-global context